sdk/core/core-tracing/src/interfaces.ts

```diff
@@ -1,10 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
 /**
  * Shorthand enum for common traceFlags values inside SpanContext
  */
-export const enum TraceFlags {
```

Comment:
please do not remove this, this lets us avoid a runtime dependency on a value that can be inlined!

Reply:
This is automatically removed by `eslint`.  Seems like it believes `enum TraceFlags` is the same as `const enum TraceFlags`. Should we document that `const` here is necessary?

Reply:
This comes from the rule `@azure/azure-sdk/ts-no-const-enums`. I will ignore this rule here.

Reply:
Let's disable that rule globally. It's madness to ban const enums.

Reply:
@willmtemple ^

Reply:
JFYI this is in relation to a guideline rule: https://azure.github.io/azure-sdk/typescript_design.html#ts-no-const-enums

(CC @bterlson )

All of our eslint rules under the `@azure/azure-sdk` plugin are based on a specific guideline entry.

Reply:
Yeah, I was thinking about this one. 

I get why it's important most of the time for JS customers, but for this specific one I was trying to type-erase the enum and inline the value since a runtime dependency on the type was causing problems and causing us to sometimes put literal 0's and 1's in our code with comments, which seemed worse.

Reply:
In a better world this would be a string union with meaningful strings :(